src/line-chart/LineChart.tsx

```diff
@@ -231,6 +231,10 @@ class LineChart extends AbstractChart<LineChartProps, LineChartState> {
     return (dataset.color || this.props.chartConfig.color)(opacity);
   };
 
+  getShadowColor = (dataset: Dataset, opacity: number) => {
+    return (dataset.shadowColor || dataset.color || this.props.chartConfig.color)(opacity);
```

Comment:
I feel like it should default to `url(#fillShadowGradient`..., to make this change backwards compatible